src/routes/goals/handlers.js

```diff
@@ -362,6 +362,7 @@ export async function getGoalHistory(req, res) {
         goalTemplateId: goal.goalTemplateId,
         grantId: goal.grantId,
         prestandard: goal.prestandard,
+        onApprovedAR: true,
```

Comment:
Is there ever a case we would show if its not on approved AR but createdVia = rtr?